modules/ipi-install-setting-cluster-node-hostnames-dhcp.adoc

```diff
@@ -6,13 +6,11 @@
 [id="ipi-install-setting-cluster-node-hostnames-dhcp_{context}"]
 = Setting the cluster node hostnames through DHCP
 
-On {op-system-first} machines, hostnames are is set through `NetworkManager`. 
-By default, the machines obtain their hostnames through DHCP. 
-If hostnames are not provided by DHCP, set statically through kernel arguments, or another method, they are obtained through a reverse DNS lookup. 
-Reverse DNS lookup occurs after the network has been initialized on a node and can take time to resolve. 
-Other system services can start prior to this and detect hostnames as `localhost` or similar. 
-You can avoid this delay in assigning hostnames by using DHCP to provide the hostname for each cluster node.
-
-Additionally, setting the hostnames through DHCP can bypass any manual DNS record name configuration errors in environments that have a DNS split-horizon implementation.
+On {op-system-first} machines, `NetworkManager` sets the hostnames. By default, DHCP provides the hostnames to `NetworkManager`, which is the recommended method. If DHCP does not provide the hostnames, if you use kernel arguments to set the hostnames, or if you use another method to set the hostnames, `NetworkManager` gets the hostnames through a reverse DNS lookup. 
```

Comment:
> If DHCP does not provide the hostnames, if you use kernel arguments to set the hostnames, or if you use another method to set the hostnames, `NetworkManager` gets the hostnames through a reverse DNS lookup.

 I was wondering if this info can use bullets. Totally up to you!

```
NetworkManager gets the hostnames through a reverse DNS lookup in the following cases:

- If DHCP does not provide the hostnames
- If you use kernel arguments to set the hostnames
- If you use another method to set the hostnames
```

Reply:
Thanks. I will modify it accordingly.